Scripts~/windows/.export-server-sdk.ps1

```diff
@@ -5,42 +5,43 @@ param(
 	[string] $ServerSdkVersion
 )
 
-$ROOT_DIR='.'
-$SERVER_SDK_FILENAME="GameLift-CSharp-ServerSDK-UnityPlugin-$ServerSdkVersion.zip";
-$DESTINATION_PATH="$ROOT_DIR\$SERVER_SDK_FILENAME"
-$TEMP_PATH="C:\Temp"
-$TEMP_ZIP_PATH = "$TEMP_PATH\GameLiftServerSDK.zip"
-$TEMP_EXTRACTED_PATH = "$TEMP_PATH\GameLiftServerSDK"
-$S3_SERVER_SDK_BUCKET="https://gamelift-server-sdk-release.s3.us-west-2.amazonaws.com/unity"
-$S3_SERVER_SDK_DOWNLOAD_LINK="$S3_SERVER_SDK_BUCKET/$SERVER_SDK_FILENAME"
-
-& "$PSScriptRoot\.verify-working-directory.ps1"
-if ($LASTEXITCODE -ne 0) { exit $LASTEXITCODE }
+$ROOT_DIR=Resolve-Path "$PSScriptRoot\..\.."
+$BUILD_DIR="$ROOT_DIR\build"
+$SERVER_SDK_FILENAME="GameLift-CSharp-ServerSDK-UnityPlugin-$ServerSdkVersion";
```

Comment:
It might make sense to make a different variable that doesn't include the ".zip" and keep $SERVER_SDK_FILENAME to include the ".zip" so you stay consistent with the other scripts.

Reply:
Updated